src/UI/Art/SpriteArtist/SpriteMenuArtist.java

```diff
@@ -17,14 +18,14 @@
   private ArrayList<String> myCurrentMenuStrings;
   private int drawScale;
 
-  private final Color MENUFRAMECOLOR = new Color(169, 118, 65);
-  private final Color MENUBGCOLOR = new Color(234, 204, 154);
-  private final Color MENUHIGHLIGHTCOLOR = new Color(246, 234, 210);
+  public static final Color MENUFRAMECOLOR = new Color(169, 118, 65);
+  public static final Color MENUBGCOLOR = new Color(234, 204, 154);
+  public static final Color MENUHIGHLIGHTCOLOR = new Color(246, 234, 210);
 
-  private int menuTextWidth;
-  private int menuTextHeight;
-  private int menuHBuffer; // Amount of visible menu to left and right of options;
-  private int menuVBuffer; // Amount of visible menu above and below menu options;
+  public static int menuTextWidth;
+  public static int menuTextHeight;
+  public static int menuHBuffer; // Amount of visible menu to left and right of options;
+  public static int menuVBuffer; // Amount of visible menu above and below menu options;
```

Comment:
Same objection to making all the values here `static`, especially since some of them can be accessed/inferred in a static way already from elsewhere.

For colors in particular, I wonder if it would be worth making a static color repository for that stuff (possibly to include team colors). Probably more than is reasonable to bite off for this change.